docs/how-to/resolve-github-connection-errors.md

```diff
@@ -0,0 +1,47 @@
+---
+id: resolve-github-connection-errors
+title: How to Resolve GitHub Connection Errors in Amplication
+sidebar_label: How To Resolve GitHub Connection Errors
+slug: /how-to/resolve-github-connection-errors
+pagination_next: how-to/change-base-branch-for-pull-requests
+---
+
+# How to Resolve GitHub Connection Errors in Amplication
+
+If you encounter an internal server error while attempting to select a repository or change the repository that your service is currently connected to in Amplication, follow these steps to resolve the issue.
+
+## Step 1: Uninstall the Amplication GitHub App
+
+1. Go to your GitHub settings by visiting [https://github.com/settings/applications](https://github.com/settings/applications).
```

Comment:
Let's mention also the relevant tab - "Installed GitHub Apps"

<img width="1019" alt="image" src="https://github.com/amplication/docs/assets/149934977/989f869b-8ba5-49f5-9928-f66c9c633b4a">